seshdash/views.py

```diff
@@ -575,6 +574,28 @@ def get_alerts(request):
 
     return HttpResponse(json.dumps(alert_data))
 
+    
+
+
```

Comment:
this requires the @login_required decorator as anyone that has logged in can only view alerts